### PR TITLE
[PAYARA-1605] NewRelic notifier documentation

### PR DESCRIPTION
**Pull request by fturizo**
Implemented documentation article for the NewRelic notifier. Here's a preview of it:

https://fturizo.gitbooks.io/payara-server-documentation/content/v/PAYARA-1605/documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.html

### CODE REVIEW

File: documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+[source, shell]
+----
+asadmin > get-newrelic-notifier-configuration
+
+Enabled  Key                               Account Id
+true     b5815wdxj6lF_tmMBljQa5y1603JTiLh  1658989
+----
+
+[[on-the-domain.xml-configuration-file]]
+=== On the _domain.xml_ configuration file
```
Comment (mikecroft) [inline documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc]: > **In** the _domain.xml_ configuration file

File: documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
++
+NOTE: NewRelic offers 14-days trial accounts that  can be used to test the service
+for free.
+
+. Configure application performance monitoring (APM) for a Payara Server instance
+by installing a New Relic Java agent
+. Configure NewRelic Insights API key that will be used to configure the notifier.
+
+[[configure-apm]]
+=== Configure Aplication Performance Monitoring for Payara Server
```
Comment (fturizo) [inline documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc]: Changed

File: documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+. First, proceed to setup the NewRelic Java agent in Payara Server. Login with
```
Comment (fturizo) [inline documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc]: Changed. Thanks for the recommendation

File: documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+    app_name: Payara Server 4.1.2.172
+
+    # ...
+----
++
+NOTE: We also recommend setting the name of the application under the `app_name`
+property to correctly identify this server's domain on the APM dashboard
+
++
+IMPORTANT: The License Key value must be inserted using single quotes ('')
```
Comment (fturizo) [inline documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc]: Changed.

File: documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+
+. Now, proceed to install the agent by running the following command at the root
```
Comment (fturizo) [inline documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc]: Added a tip on how to install the agent for standalone instances and clusters.

File: documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+editing the _domain.xml_ configuration file directly.
+
+[[using-the-administration-web-console]]
+=== Using the Administration Web Console
+
+To configure the Notification Service in the Administration Console, go
+to _Configuration -> [instance-configuration (like server-config)] ->
+Notification Service_ and click on the *NewRelic* tab:
+
+image:/images/notification-service/newrelic/admin-console-configuration.png[Slack Configuration on Admin Console]
```
Comment (fturizo) [inline documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc]: Changed the layout a bit to make it more readable.

File: documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc (hunk 1 above)
Comment (fturizo) [inline documentation/extended-documentation/notification-service/notifiers/newrelic-notifier.adoc]: Changed

**Pull request merged**